src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
Strange. Mine tests show only one media break on 960px, which is correct. Maybe update your deps with npm install?